mozilla/fx-private-relay
# Add announcement area for new features

# New feature description

This adds a menu that highlights new features. Note that this is slightly limited compared to the acceptance criteria: since only a few entries are shown at this time, the menu shows the entire history. I've prepared a separate PR to implement the full feature set in React, which will be used by the time there are more entries: https://github.com/mozilla/fx-private-relay/pull/1562.

# Screenshot (if applicable)

https://user-images.githubusercontent.com/4251/154987854-dce94cf3-54d6-4b11-888c-3ba99f22269d.mp4

# How to test

- Sign in with a pro account and with the add-on enabled.
- See that there's a "News" entry in the menu, with a counter set to 3.
- Verify that the counter is decremented when opening one of the entries for the first time.
- With different accounts, verify that the counter is reset again. Additionally:
- With the extension not installed, verify that there is no entry for signing back in.
- Without Premium, verify that there is no entry about critical emails.

# Checklist

- [ ] l10n dependencies have been merged, if any. - Not yet: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/63
- [ ] All acceptance criteria are met. - No, see above.
